docs/_deploying/heroku.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
```

Comment:
I'll open a team discussion to talk about where the example apps should live - separate repository, subfolder in bolt, or elsewhere. Afterwards, I'll update the links in this guide.

Reply:
Added the _Getting Started with Heroku_ app to the `examples/` directory. Commit e34e905